Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN dotnet build
 RUN dotnet publish --runtime alpine-x64 -c Release -o out --self-contained true /p:PublishTrimmed=true
 
 # build runtime image with DoD CA Certificates
-FROM docker.io/alpine:3.18.4
+FROM ghcr.io/soteriasoftwarellc/openrmfpro-base:2.10.1
```

Comment:
This is a private image, so that will not work. Need to use the updated openrmf-base image here